azure-pipelines/test.yml

```diff
@@ -51,16 +46,28 @@ steps:
     mergeTestResults: true
     testRunTitle: $(System.StageDisplayName) $(Agent.JobName) (Attempt $(System.JobAttempt))
 
+- powershell: |
```

Comment:
The template parameter `isIntegration` is sourced back to a variable in test-matrix.yml. This makes it unusable for `${{ if }}` statements. So instead of excluding steps based on a template condition, I will set a variable which can be used to conditionally run the integration log publishing steps.